lib/runner.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
+var Promise = require('pinkie-promise');
```

Comment:
Actually. I think it's worth using https://github.com/petkaantonov/bluebird here for performance reasons.